tests/test_metrics.py

```diff
@@ -163,6 +178,16 @@ def testSkySources(self):
         self.assertEqual(meas.metric_name, Name(metric="ip_diffim.fracDiaSourcesToSciSources"))
         self.assertEqual(meas.quantity, len(diaCatalog) / (len(sciCatalog) - 1) * u.dimensionless_unscaled)
 
+    def testPrimarySources(self):
+        sciCatalog = _makeDummyCatalog(5, skyFlag=True, priFlag=True)
+        diaCatalog = _makeDummyCatalog(3)
+        result = self.task.run(sciCatalog, diaCatalog)
+        lsst.pipe.base.testUtils.assertValidOutput(self.task, result)
+        meas = result.measurement
+
+        self.assertEqual(meas.metric_name, Name(metric="ip_diffim.fracDiaSourcesToSciSources"))
+        self.assertEqual(meas.quantity, len(diaCatalog) * u.dimensionless_unscaled)
```

Comment:
Also at lines 112, 179: All `Quantities` are dtype float. Is it safe to use `assertEqual` (instead of `assertFloatsAlmostEqual` or similar) in general?